cmd/nknc/wallet/wallet.go

```diff
@@ -79,29 +78,57 @@ func walletAction(c *cli.Context) error {
 	// create wallet
 	if c.Bool("create") {
 		if common.FileExisted(name) {
-			fmt.Printf("CAUTION: '%s' already exists!\n", name)
-			os.Exit(1)
-		} else {
-			wallet, err := vault.NewWallet(name, getConfirmedPassword(passwd))
-			if err != nil {
-				fmt.Fprintln(os.Stderr, err)
-				os.Exit(1)
+			return fmt.Errorf("CAUTION: '%s' already exists!\n", name)
+		}
+		wallet, err := vault.NewWallet(name, getConfirmedPassword(passwd))
+		if err != nil {
+			return err
+		}
+		showAccountInfo(wallet, false)
+		return nil
+	}
+
+	var hexFmt bool
+	switch format := c.String("restore"); format {
+	case "": // Not restore mode
+		break
+	case "hex":
+		hexFmt = true
+		fallthrough
+	case "bin":
+		if common.FileExisted(name) {
+			return fmt.Errorf("CAUTION: '%s' already exists!\n", name)
+		}
+
+		key, err := password.GetPassword("Input you PrivateKey")
```

Comment:
Should be seed instead of PrivateKey

Reply:
I worry about that user will confuse with crypto seed(nknc) & DHT node seed(nknd).

Reply:
Then we should use `Secret Seed`. Private key is different from seed in Ed25519

Reply:
Done.